packages/@lwc/ssr-compiler/src/__tests__/utils/expected-failures.ts

```diff
@@ -18,9 +18,9 @@ export const expectedFailures = new Set([
     'adjacent-text-nodes/with-comments/preserve-comments2/index.js',
     'attribute-aria/dynamic/index.js',
     'attribute-class/unstyled/dynamic/index.js',
-    'attribute-class/unstyled/static/index.js',
+    'attribute-class/with-scoped-styles-only-in-child/dynamic/index.js',
+    'attribute-class/with-scoped-styles-only-in-parent/dynamic/index.js',
     'attribute-class/with-scoped-styles/dynamic/index.js',
-    'attribute-class/with-scoped-styles/static/index.js',
```

Comment:
We need to implement template class binding: https://github.com/salesforce/lwc-rfcs/pull/87

Reply:
https://github.com/salesforce/lwc/issues/4839